bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -599,6 +635,7 @@ type KubeConfigCluster struct {
 	// Defaults to the Cluster's CA certificate if empty.
 	//
 	// +optional
+	// +kubebuilder:validation:MaxLength=4096
```

Comment:
I will increase this in case someone uses a bundle with more than one certificate

Reply:
To how much? I used 51200 (50kb) for now

This is probably already enough to run into problems with user data (there is a size limit in some providers)